src/main/java/joe/ui/Ui.java

```diff
@@ -0,0 +1,112 @@
+package joe.ui;
+
+import java.util.ArrayList;
+
+import joe.task.Task;
+
+public class Ui {
+    public static final String HORIZONTAL_LINE = "____________________________________________________________";
+    public static final String ADD_TASK_MESSAGE = "Got it. I've added this task:\n";
+    public static final String TASK_COUNT_MESSAGE = "Now you have %d tasks in the list.\n";
+
+    private final String CHATBOT_NAME;
+
+    public Ui(String chatbotName) {
+        this.CHATBOT_NAME = chatbotName;
+    }
+
+    public void greet() {
+        System.out.printf(
+                "%s\nHello! I'm %s\nWhat can I do for you?\n%s\n",
+                HORIZONTAL_LINE,
+                CHATBOT_NAME,
+                HORIZONTAL_LINE);
+    }
+
+    public void farewell() {
+        System.out.printf("Bye. Hope to see you again soon!\n%s", HORIZONTAL_LINE);
+    }
+
+    public void printHelp() {
+        System.out.println("Here are the commands you can use:");
```

Comment:
Should this be formatted before print so you won't have so many System.out.println?